### PR TITLE
Make it build again

### PR DESCRIPTION
**Pull request by vext01**
This improves the build system and makes the interpreter build again.

### CODE REVIEW

**Comment by vext01** [comment]
I didn't realise this was a bors repo. I will need to edit `.buildbot.sh`.

In the meantime, please go ahead and review what is already here.

**Comment by ltratt** [comment]
I was just about to `r+` so please push a new commit for buildbot.sh.

**Comment by vext01** [comment]
There are also tests that I need to see if I can recoup. They won't be aware of guard failures.

I wonder if the `yk` repo's CI should be running the tests in this repo too...

**Comment by ltratt** [comment]
> I wonder if the yk repo's CI should be running the tests in this repo too...

Possibly, but I think that's not something we should worry about in this PR: ykcbf can stand on its own and separately be used by yk/ykllvm.

**Comment by vext01** [comment]
This works for now.

Note that the tests don't test the JIT yet. We won't be able to do that for a bit, so there's no point in enabling the JIT just yet.

File: .buildbot.sh [hunk 1]
```diff
@@ -29,13 +27,14 @@ make -j `nproc` install
 export PATH=`pwd`/../inst/bin:${PATH}
 cd ../..
 
+
```
Comment (ltratt) [inline .buildbot.sh]: Two blank lines?

Comment (vext01) [reply .buildbot.sh]: Fixed in 1429017

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**